### DISCUSSION TITLE
CCS workaround

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#4481] [Unsupported Versions] [answered]
As per [this](https://github.com/elastic/elasticsearch/issues/64784) issue ELK does not support cross-cluster search (CCS) due to this I can't create EQL queries on SIEM detection as it considers `*:so-*` index invalid. Can you suggest any workaround here so that my queries will work? Like using only `so-*` index?

**Comment by weslambert** [comment, marked as answer]
You could certainly try that -- however, you would need to be using standalone mode, a single search node, or using advanced mode (which operates as a single cluster).
